packages/typography/src/components/body-text/body-text.component.tsx

```diff
@@ -20,6 +21,8 @@ type Props = {
   lineHeight?: LineHeight;
   /** Color */
   color?: string;
+  /** Text overflow */
+  isTextOverflow?: boolean;
```

Comment:
I believe the property name should be named more like `textEllipsis: boolean` - as `isTextOverflow` indicates that text overflows on not.

Reply:
what do you think about `enableTextEllipsis`?

Reply:
Yes, IMO it's much better and self-descriptive.